src/ReachSets/ContinuousPost/BFFPSV18/BFFPSV18.jl

```diff
@@ -59,49 +56,14 @@ function options_BFFPSV18()
                  "predicate over indices; the default corresponds to ``-1``"),
 
         # approximation options
-        OptionSpec(:block_types, nothing, domain=Union{Nothing,
-            Dict{Type{<:LazySet}, AbstractVector{<:AbstractVector{Int}}}},
-            info="short hand to set ':block_types_init' and " *
-                 "':block_types_iter'"),
-        OptionSpec(:block_types_init, nothing, domain=Union{Nothing,
-            Dict{Type{<:LazySet}, AbstractVector{<:AbstractVector{Int}}}},
-            info="set type for the approximation of the initial states for " *
-                 "each block"),
-        OptionSpec(:block_types_iter, nothing, domain=Union{Nothing,
-            Dict{Type{<:LazySet}, AbstractVector{<:AbstractVector{Int}}}},
-            info="set type for the approximation of the states ``X_k``, " *
-                 "``k>0``, for each block"),
-        OptionSpec(:ε, Inf, domain=Float64, domain_check=(v  ->  v > 0.),
-            info="short hand to set `:ε_init` and `:ε_iter`"),
-        OptionSpec(:ε_init, Inf, domain=Float64, domain_check=(v  ->  v > 0.),
-            info="error bound for the approximation of the initial states" *
+        OptionSpec(:block_options, nothing, domain=Any,
```

Comment:
Should we add a domain here? From [this function](https://github.com/JuliaReach/LazySets.jl/blob/master/src/Approximations/decompositions.jl#L235) and the [notes](https://github.com/JuliaReach/LazySets.jl/blob/master/src/Approximations/decompositions.jl#L97)

```julia
domain=Union{Type{<:LazySet},
                                        Pair{<:UnionAll, <:Real},
                                        Real,
                                        Type{<:AbstractDirections}
}
```

Reply:
No, because there is also [this function](https://github.com/JuliaReach/LazySets.jl/blob/8bc5c0bafbd9e1d92c0640653c8584e17994336f/src/Approximations/decompositions.jl#L221) whose type is intentially not specified further (to allow for example to pass a vector or a dictionary).

Reply:
We could add `Vector` and `Dict` to the `Union`, but ok. Probably better to annotate the type of that function in lazysets in the first place.

Reply:
I've opened https://github.com/JuliaReach/LazySets.jl/issues/1202 as follow up.